source/content/guides/decoupled/drupal-backend-starters/03-local-dev.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Decoupled Drupal Backend Starter for Front-End Sites
+subtitle: Local Development
+description: Learn how to set up your local development environment.
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia, whitneymeredith]
+layout: guide
+permalink: docs/guides/decoupled/drupal-backend-starters/local-dev
+anchorid: local-dev
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides information on how to set up local development for your project.
+
+## Before You Begin
+
+The example local environment configurations outlined below are based on
+[Lando](https://lando.dev/).
+
+- [Lando installation instructions](https://docs.lando.dev/getting-started/installation.html).
+
+- Depending on your role, you may only need a local backend environment, or you may prefer a local environment that can run both the backend and
+frontend. The following sections outline the steps to set up a local
+environment for each of these use cases.
+
+## Backend Only Setup
+
+You can use the default Pantheon recipe provided by Lando if you are creating a local environment for your Drupal backend only. Refer to the [Getting Started section of the Lando Pantheon Plugin documentation](https://docs.lando.dev/pantheon/getting-started.html) for more information.
+
+## Combined Frontend and Backend Setup
+
+1. Create a `.lando.yml` file in the root of the project.
+
+1. Copy the following config into `.lando.yml`. Note: Replace `%SITE-NAME%`, `%PANTHEON_SITE_ID%`, and `%PORT-NUMBER%`, with the site name, site id, and required port number for the Front-End Site.
```

Comment:
@whitneymeredith have you tested these instructions? How would the developer know what to use as PORT-NUMBER?

Reply:
@stevector All technical steps and contextual content was taken from the decoupledkit. pantheon.io site vetted and maintained by Brian and Coby.

Reply:
@CobyPear @backlineint How does the user know what port number to use?

Reply:
The port number will vary based on the JS framework you are using. Next uses 3000. Gatsby uses 8000. They can be configured to use other ports. Other frameworks may use other ports.

I wasn't expecting these instructions to be ported to the platform docs. To be honest, I don't think they are mature enough for that. I think we would need a better solution than copying and pasting a large config file like this.